P1684-mdarray/P1684.md

```diff
@@ -22,6 +22,13 @@ author:
 #### Changes from R1
 
 - update and reword non-wording text to harmonize with P0009R16
+- fix synopsis missing Alloc argument in some places
+- fix constraints on some constructors to allow std::array as container
+- add missing wording for std::initializer list
+- add constructors from ranges
```

Comment:
They asked for this? :/

Reply:
more or less, they asked what about those, and both container adaptors and containers have them so ...